inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceTest.java

```diff
@@ -0,0 +1,264 @@
+        kb.setType(RepositoryType.REMOTE);
+        sut.registerKnowledgeBase(kb, sut.getRemoteConfig(profile.getSparqlUrl()));
+        setSchema(kb, OWL.CLASS, RDFS.SUBCLASSOF, RDF.TYPE, RDFS.COMMENT, RDFS.LABEL, RDF.PROPERTY);
+        List<KBHandle> propertiesKBHandle = sut.listProperties(kb, mapping.getPropertyTypeIri(), true, true);
+        log.debug("\n \n Size of List Properties DBPedia ::::::::" + propertiesKBHandle.size() + "\n \n");
+        assertThat(propertiesKBHandle).as("Check that list is not empty").isNotEmpty();
+
+    }
+
+    @Test
+    public void testListPropertiesKnowledgeBaseIRI_WithYago() throws IOException
+    {   
+        String kbProfile = "yago";
+        KnowledgeBaseProfile  profile = sut.readKnowledgeBaseProfiles().get(kbProfile);
+        KnowledgeBaseMapping mapping = profile.getMapping();
+        kb = buildKnowledgeBase(project, KB_NAME.concat("-" + kbProfile));
+        kb.setType(RepositoryType.REMOTE);
+        sut.registerKnowledgeBase(kb, sut.getRemoteConfig(profile.getSparqlUrl()));
+        setSchema(kb, OWL.CLASS, RDFS.SUBCLASSOF, RDF.TYPE, RDFS.COMMENT, RDFS.LABEL, RDF.PROPERTY);
+        List<KBHandle> propertiesKBHandle = sut.listProperties(kb, mapping.getPropertyTypeIri(), true, true);
+        log.debug("\n \n Size of List Properties Yago ::::::::"
+                + propertiesKBHandle.size() + "\n \n");
+        assertThat(propertiesKBHandle).as("Check that list is not empty").isNotEmpty();
+
+    }
+
+    // Helper
+    
+    private KnowledgeBase buildKnowledgeBase(Project project, String name) {
+        KnowledgeBase kb = new KnowledgeBase();
+        kb.setName(name);
+        kb.setProject(project);
+        kb.setType(RepositoryType.LOCAL);
+        kb.setClassIri(RDFS.CLASS);
+        kb.setSubclassIri(RDFS.SUBCLASSOF);
+        kb.setTypeIri(RDF.TYPE);
+        kb.setLabelIri(RDFS.LABEL);
+        kb.setPropertyTypeIri(RDF.PROPERTY);
+        kb.setDescriptionIri(RDFS.COMMENT);
+
+        kb.setReification(reification);
+        return kb;
+       
+    }
+
+    private void importKnowledgeBase(String resourceName) throws Exception {
+        ClassLoader classLoader = getClass().getClassLoader();
+        String fileName = classLoader.getResource(resourceName).getFile();
+        try (InputStream is = classLoader.getResourceAsStream(resourceName)) {
+            sut.importData(kb, fileName, is);
+        }
+    }
+    
+    private void setSchema(KnowledgeBase kb, IRI classIri, IRI subclassIri, IRI typeIri,
+        IRI descriptionIri, IRI labelIri, IRI propertyTypeIri) {
+        kb.setClassIri(classIri);
+        kb.setSubclassIri(subclassIri);
+        kb.setTypeIri(typeIri);
+        kb.setDescriptionIri(descriptionIri);
+        kb.setLabelIri(labelIri);
+        kb.setPropertyTypeIri(propertyTypeIri);
+        kb.setReification(reification);
+        sut.updateKnowledgeBase(kb, sut.getKnowledgeBaseConfig(kb));
+    }
+    
```

Comment:
Whitespace

Reply:
I am removing the class, as it has been already changed to https://github.com/inception-project/inception/blob/473f67a64f1fef3fd47db7cca6e8a95d78ee2989/inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceRemoteTest.java#L70